python/publish/handle_following_version.py

```diff
@@ -0,0 +1,21 @@
+import sys
+from packaging.version import Version
+
+version = Version(sys.argv[1])
+new_version = None
+
+if version.is_devrelease:
+    # For dev releases, increment the dev release number.
+    new_version = f"{version.major}.{version.minor}.{version.micro}.dev{version.dev + 1}"
+elif version.is_prerelease:
+    # For pre releases, go back to dev release.
+    rel_type, rel_num = version.pre
```

Comment:
Are these variables used?

Reply:
Good catch!

Reply:
I don't see any changes pushed, should there be?

Reply:
lol I didn't save the file